pkg/apis/imagepolicy/types.go

```diff
@@ -64,4 +64,9 @@ type ImageReviewStatus struct {
 	// may contain a short description of what is wrong.  Kubernetes
 	// may truncate excessively long errors when displaying to the user.
 	Reason string
+	// Annotations should be added to the annotations of the pod. They
+	// may be used to later find pods, e.g. ones that require audits.
+	// Annotations may only be set in case 'Allowed' is True.  They should
+	// match the pattern '*.image-policy.k8s.io/*'.
+	Annotations map[string]string
```

Comment:
Should these annotations be added to the pods themselves, or just the audit events (now that we support that)?

Either way, the field name should probably reflect that.

Reply:
I'd prefer the pods themselves as then they will be picked up by future audits (e.g., by future changes in Kritis).